lib/solarwinds_apm/patch/tag_sql/sw_dbo_utils.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module SolarWindsAPM
+  module Patch
+    module TagSql
+      module SWODboUtils
+        def self.annotate_span_and_sql(sql)
+          return sql if sql.to_s.empty?
+
+          current_span = ::OpenTelemetry::Trace.current_span
+
+          annotated_sql = ''
+          if current_span.context.trace_flags.sampled?
+            traceparent = SolarWindsAPM::Utils.traceparent_from_context(current_span.context)
+            annotated_traceparent = "/*traceparent='#{traceparent}'*/"
+            current_span.add_attributes({ 'sw.query_tag' => annotated_traceparent })
+            annotated_sql = "#{sql} #{annotated_traceparent}"
```

Comment:
it looks the order matters?

Reply:
line 20 and 21?

Reply:
this ` annotated_sql = "#{sql} #{annotated_traceparent}"` vs `  annotated_sql = "#{annotated_traceparent} #{sql}"